styles/styles.css

```diff
@@ -0,0 +1,153 @@
+@import url("./fonts.css");
+
+* {
+  margin: 0 auto;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  box-sizing: border-box;
+  max-width: 1440px;
+  height: 100vh;
+}
+
+.top-section {
+  width: 1440px;
+  height: 900px;
+  justify-content: space-between;
+  padding-top: 47px;
+  background: url(../images/castelmezzano-background-background.png) no-repeat center;
+  background-size: cover;
+  color: #ffffff;
+}
+
+.header {
+  display: flex;
+ }
+
+.logo-img {
+ background-image: url("../images/sprite.png");
+  width: 205px;
+  height: 50px;
+  margin-left: 104px;
+  margin-right: 583px;
+}
+
+.item-1 {
```

Comment:
Unclear name.

Reply:
Proper names from BEM I'm changing in 7th lesson